### DISCUSSION TITLE
OSC height fix

### DISCUSSION THREAD

**Discussion by Keith94** [Samillion/ModernZ#222] [General]
On line 1566 the OSC height is set to 180. I don't think that's a good value because it causes my issue #20.

After a short amount of testing, I found these work best, at least for me.

h = 108 when showtitle=no
h = 155 when showtitle=yes

With these values, now any mouse commands like right click are working perfectly above OSC area.

Any chance for a similar fix like that?

**Comment by Samillion** [comment]
> Any chance for a similar fix like that?

Of course! If it's not a bother, could you make a feature request issue with it? To keep track properly.

**Reply by Keith94** [reply]
Ye. Thanks man. #223
